uploadservice/src/main/java/net/gotev/uploadservice/UploadService.kt

```diff
@@ -194,6 +195,14 @@ class UploadService : Service() {
 
         wakeLock = acquirePartialWakeLock(wakeLock, TAG)
         notificationActionsObserver.register()
+
+        val notification = NotificationCompat.Builder(this, UploadServiceConfig.defaultNotificationChannel!!)
+            .setSmallIcon(android.R.drawable.ic_menu_upload)
+            .setOngoing(true)
+            .setGroup(UploadServiceConfig.namespace)
+            .build()
+
+        startForeground(UPLOAD_NOTIFICATION_BASE_ID, notification)
```

Comment:
why are we making changes to the core library? this should be upstream and done via the main repo/project..? we want to be able to rebase their changes easily?

Reply:
This is cherry picked from the author's new commits.